src/toil_lib/tools/QC.py

```diff
@@ -27,3 +27,33 @@ def run_fastqc(job, r1_id, r2_id):
     output_files = [os.path.join(work_dir, x) for x in output_names]
     tarball_files(tar_name='fastqc.tar.gz', file_paths=output_files, output_dir=work_dir)
     return job.fileStore.writeGlobalFile(os.path.join(work_dir, 'fastqc.tar.gz'))
+
+
+def run_bam_qc(job, aligned_bam_id, uuid=None):
```

Comment:
Practically, I doubt this will be used elsewhere but there isn't any reason someone couldn't run this in another pipeline. Happy to close this and move it the toil-rnaseq project.  Would it be fine to put it in an external module to the toil-rnaseq source?